src/alcaramba/Drawable/Town/Town.gd

```diff
@@ -86,9 +86,30 @@ func remove_tile(x: int, y: int) -> int:
 	return tile_id
 
 # check if removing this tile would break continouity of town
-func is_tile_removable(x: int, y: int):
-	# TODO #15: logic for continuity
-	return true
+func _is_tile_removable(x: int, y: int):
+	
+	var removal_valid = true
+	
+	# if tile is not populated or starting tile, removal not valid
+	if get_cell(x, y) == TileMap.INVALID_CELL || get_cell(x, y) == _starting_tile_id: return false
+	
+	# loop over each neighbour, remove test tile and neighbour then check if neighbour placement is stall valid
+	var test_id = get_cell(x, y)
+	set_cell(x, y, TileMap.INVALID_CELL)
+	for _x in range(-1, 2):
```

Comment:
This is not enough. You need to check all the way back to the start tile, as everything needs to be connected back to it.

Reply:
![alcaramba-2022-12-29-15-continuity](https://user-images.githubusercontent.com/29922/210012553-c59be987-4bc2-4a4b-ba41-f30922f1feb4.gif)